src/components/organism/SwapTokens/index.tsx

```diff
@@ -104,87 +339,226 @@ const SwapTokens = () => {
+              className={`flex basis-1/2 justify-center rounded-lg  px-4 py-3 ${
+                slippageAuto ? "bg-purple-100" : "bg-white"
+              }`}
+              onClick={() => {
+                setSlippageAuto(true);
+                setSlippageValue(10);
+              }}
+            >
+              {t("tokenAmountInput.auto")}
+            </button>
+            <button
+              className={`flex basis-1/2 justify-center rounded-lg px-4 py-3 ${
+                slippageAuto ? "bg-white" : "bg-purple-100"
+              }`}
+              onClick={() => setSlippageAuto(false)}
+            >
+              {t("tokenAmountInput.custom")}
+            </button>
           </div>
-          <div className="flex flex-row gap-2">
-            <div className="flex w-full basis-8/12 flex-row rounded-xl bg-white p-1 text-large font-normal text-text-color-header-light">
-              <button
-                className={`flex basis-1/2 justify-center rounded-lg  px-4 py-3 ${
-                  slippageAuto ? "bg-purple-100" : "bg-white"
-                }`}
-                onClick={() => {
-                  setSlippageAuto(true);
-                  setSlippageValue(15);
-                }}
-              >
-                {t("tokenAmountInput.auto")}
-              </button>
-              <button
-                className={`flex basis-1/2 justify-center rounded-lg px-4 py-3 ${
-                  slippageAuto ? "bg-white" : "bg-purple-100"
-                }`}
-                onClick={() => setSlippageAuto(false)}
-              >
-                {t("tokenAmountInput.custom")}
-              </button>
-            </div>
-            <div className="flex basis-1/3">
-              <div className="relative flex">
-                <NumericFormat
-                  value={slippageValue}
-                  onValueChange={({ floatValue }) => setSlippageValue(floatValue)}
-                  fixedDecimalScale={true}
-                  thousandSeparator={false}
-                  allowNegative={false}
-                  className="w-full rounded-lg bg-purple-100 p-2 text-large  text-text-color-label-light outline-none"
-                  placeholder="15"
-                  disabled={slippageAuto ? true : false}
-                />
-                <span className="absolute bottom-1/3 right-2 text-medium text-text-color-disabled-light">%</span>
-              </div>
+          <div className="flex basis-1/3">
+            <div className="relative flex">
+              <NumericFormat
+                value={slippageValue}
+                onValueChange={({ value }) => setSlippageValue(parseInt(value) >= 0 ? parseInt(value) : 0)}
+                fixedDecimalScale={true}
+                thousandSeparator={false}
+                allowNegative={false}
+                className="w-full rounded-lg bg-purple-100 p-2 text-large  text-text-color-label-light outline-none"
+                disabled={slippageAuto ? true : false}
```

Comment:
Please simplify this condition, you can put just disabled={slippageAuto}

Reply:
done